PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,111 @@
+# Description
+> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context and any links to documentation created.
+> Reference: [PR Submission Guide](/submission_process.md)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
idt they need to delete them. theres not that many i think its ok (maybe better?) to have it explicitly show that you didn't tick a certain box